docs/contributor/03-11-hap-rules.md

```diff
@@ -0,0 +1,244 @@
+| Platform Region (PR) 	| string, platform regions as defined in https://help.sap.com/docs/btp/sap-business-technology-platform/regions-for-kyma-environment 	| true            	| false            	| hyperscalerType: `<providerType>_<PR>` or hyperscalerType: `<providerType>_<PR>_<HR>` if used with HR 	|
+| Hyperscaler Region (HR)  	| string, hyperscaler region as defined in https://help.sap.com/docs/btp/sap-business-technology-platform/regions-for-kyma-environment   	| true            	| false            	| hyperscalerType: `<providerType>_<HR>` or hyperscalerType: `<providerType>_<PR>_<HR>` if used with PR 	|
+| Eu Access (EU)       	| no value                                                                                                                           	| false           	| true             	| euAccess: true                                                                                    	|
+| Shared (S)           	| no value                                                                                                                           	| false           	| true             	| shared: true                                                                                      	|
+
+
+
+## Validation
+
+KEB validates HAP Rules during startup. If the configuration is invalid, KEB will not start and an error message will be displayed in the logs. 
+
+The constraints used for validation during KEB startup include:
+* Rules format check - all the rules must comply with the format specified above.
+* Every rule entry must refer to an existing plan - if you define a rule with non existing plan then an error is returned during KEB startup that fails the process.
+* Every supported plan needs at least one rule entry, if no rule entry is defined for a plan then an error is returned during KEB startup.
+* Uniqueness validation check - KEB checks if all rule entries are unique in the scope of the rule. Specifying more than one entry with the same number of identification attributes is prohibited. If more than one such attribute is specified then the error is returned failing KEB startup. For more details see [Uniqnuess & Priority](#uniqnuess--priority) section. 
+
+
+## Initial Configuration
+
+The last example shows initial configuration created to mimic the current bahaviour of KEB at the time of writing the document. The configuration enforces that:
+* azure, aws, gcp have their own pools of dedicated bindings.
+* gcp clusters in the region cf-sa30 use the pool of secret bindings marked with labels: `hyperscalerType: gcp_cf-sa30`,
+* sap-converged-cloud clusters use the pool of secret bindings marked with labels: `hyperscalerType: openstack_<HYPERSCALER_REGION>` and all these pools are shared.
+* trial clusters can use one of two pools of shared secret bindings marked with labels: `hyperscalerType: azure` or `hyperscalerType: aws` depending on the used trial provider type.
```

Comment:
I think this is hardcoded. PLease check and then change that trial has always aws.

Reply:
I will check tomorrow if it is controlled with: APP_PROVISIONER_DEFAULT_TRIAL_PROVIDER